tests/components/lcn/conftest.py

```diff
@@ -113,12 +110,21 @@ def lcn_connection_factory(*args, **kwargs):
 
     entry.add_to_hass(hass)
     with patch(
-        "pypck.connection.PchkConnectionManager",
-        side_effect=lcn_connection_factory,
+        "pypck.connection.PchkConnectionManager", side_effect=lcn_connection_factory
```

Comment:
Can we also change how we patch this? We rather have patches to `homeassistant.components.lcn.....`

Reply:
I changed the code, so  that the PchkConnectionManager is imported and then patched in the `homeassistant.components.lcn` context,
But all the other tests still patch it in the context of the imported `pypck`-library. I'd also like to address this in a separate PR.